src/core_modules/capture-core/components/Notes/Notes.component.js

```diff
@@ -35,7 +35,7 @@ const styles = theme => ({
     borderBoxContent: {
         margin: theme.typography.pxToRem(10),
     },
-    newCommentButtonContainer: {
+    newNoteButtonContainer: {
```

Comment:
@henrikmv The `newNoteButtonContainer` class is declared twice, which causes some styles to be overwritten. Can you combine all the styles into one definition?
 Thanks!

![Screenshot 2024-07-10 at 09 48 31](https://github.com/dhis2/capture-app/assets/6817073/48dea5a9-f4f6-4521-a25c-36e502b72810)

Reply:
Thank you, @simonadomnisoru! 

I could not combine the two classes because they represent two different cases. One for the case on the image over, where you have the input field and the _add note_ and _cancel_ button. And the other case is for the image below, with just the _write note_ button. I added a plural _s_ for readability for the class in the first scenario, and removed the `inline-block` class in scenario two because I could not see the class made any different behavior for the _write note_ button.
<img width="305" alt="Skjermbilde 2024-07-10 kl  12 00 38" src="https://github.com/dhis2/capture-app/assets/110386561/cf78ddf2-a5c8-4320-b715-c45e55d8bce6">